### PR TITLE
[Storage CMK Key Vault Key Auto Rotation] Add read-only properties for current versioned key in use and last rotation timestamp.

### PR DESCRIPTION
**Pull request by tonykjose**
…r current versioned key in use and last rotation timestamp.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by mmyyrroonn — APPROVED** [review]

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json [hunk 1]
```diff
@@ -1604,6 +1604,19 @@
           "type": "string",
           "description": "The Uri of KeyVault.",
           "x-ms-client-name": "KeyVaultUri"
+        },
```
Comment (blueww) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json]: Please add example for it, include create/update/get/list account.

Comment (sanjaiganesh) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json]: yes. example is required.

---
In reply to: [388050640](https://github.com/Azure/azure-rest-api-specs/pull/8585#discussion_r388050640) [](ancestors = 388050640)

Comment (MyronFanQiu) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json]: Once examples are added, I will merge this PR.

Comment (tonykjose) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json]: Added example

Comment (MyronFanQiu) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json]: @tonykjose Please fix the CI errors. PrettierCheck can be fixed by adding the word into https://github.com/Azure/azure-rest-api-specs/blob/master/custom-words.txt

Comment (tonykjose) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json]: Fixed and updated.

**Pull request merged**